Ship_Game/Universe/ExplorableGameObject.cs

```diff
@@ -9,11 +9,37 @@ public class ExplorableGameObject : GameObject
         // this is a tiny bitset where Empire.Id is the bit index for Explored:true|false
         [StarData] SmallBitSet ExploredBy;
 
+        [StarData] public bool IsResearchable { get; private set; }
```

Comment:
Lol, this single `bool` takes as much memory as the `SmallBitSet ExploredBy`. 🤡

Reply:
not sure why thats bad. we have tons of bools in the code.